src/libraries/Microsoft.Extensions.Logging/tests/Microsoft.Extensions.Logging.Generators.Tests/LoggerMessageGeneratorParserTests.cs

```diff
@@ -298,69 +298,6 @@ partial class C
             Assert.Equal(DiagnosticDescriptors.MissingRequiredType.Id, diagnostics[0].Id);
         }
 
-        [Fact]
-        public async Task MissingStringType()
```

Comment:
Why are these tests deleted?

Reply:
because this code block was removed:

![image](https://user-images.githubusercontent.com/5897654/116486495-a4bba180-a842-11eb-94d9-cf8d46fecd7c.png)

Reply:
Cool, thanks.